### PR TITLE
[MTDB-13236] [MTDB-13163] - feat(leaderboard) ✨  Leaderboard redesign pagination

### PR DESCRIPTION
**Pull request by ssotomayor**
**Short Description:**

**Figma Link (optional):**
N/A

**Screenshots (optional):**
N/A

**This PR is blocked by or a continuation of:**
PR#

### CODE REVIEW

File: src/components/Leaderboard/styled.ts [hunk 1]
```diff
@@ -5,23 +5,23 @@ import { colors } from "../colors";
 export const LeaderboardWrapper = styled.div`
   margin: 0 auto auto auto;
   width: 98%;
+  max-width: 646px;
```
Comment (PorlyBe) [inline src/components/Leaderboard/styled.ts:8]: lets make this an even 40rem (the main page is 40 i think 🤔 )

File: src/views/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  currentPage,
+  onPageChange,
+}: Props<T>): ReactNode => {
+  const totalPages = Math.ceil(data.total / pageSize);
+
+  const handlePageClick = (page: number) => {
+    onPageChange(page);
+  };
+
+  const renderPageNumbers = () => {
```
Comment (PorlyBe) [inline src/views/Pagination.tsx:104]: i dunno how i feel about this.... specificaly the mutation and for loop 🤔

**Review by PorlyBe — APPROVED** [review]
please remove those old icons... this is working well, i think it still needs some teaks but lets get this in so we can list what else needs doing

File: src/views/Pagination.tsx (hunk 2 above)
Comment (ssotomayor) [reply src/views/Pagination.tsx:104]: how would you have it? 🤔

**Pull request merged**